EpsilonLights/Src/Lights.c

```diff
@@ -27,8 +27,13 @@ void updateLightsTask(void const* arg)
         hazards = (lightsInputs >> HAZARDS_INPUT_INDEX) & 1;
 
         /* UPDATE HEADLIGHTS */
-
-        if (headlightsOff && headlightsLow + headlightsHigh)
+        if ((headlightsOff) && (headlightsLow || headlightsHigh))
+        {
+            // Error state, turn headlights off
+            HAL_GPIO_WritePin(HHIGH_GPIO_Port, HHIGH_Pin, LIGHT_OFF);
+            HAL_GPIO_WritePin(HLOW_GPIO_Port, HLOW_Pin, LIGHT_OFF);
+        }
+        else if (headlightsLow && headlightsHigh)
```

Comment:
I think this checking for an error state could be written better. Ultimately what's being checked for is that there is no more than 1 of `headlightsOff`, `headlightsLow` and `headlightsHigh` is on at any given point in time. How's it's written now doesn't feel like it's directly checking that.